.gitignore

```diff
@@ -13,3 +13,4 @@ doc/build/
 
 # development environment
 .env
+.idea
```

Comment:
It's showing up as a warning that there's no newline at the end of the file. So, we might as well keep Github happy.